### PR TITLE
Malins happy thoughts

### PR DESCRIPTION
**Pull request by malin798**
I renamed the /:thoughtsId/like to /thoughts/:thoughtId/like because I think it is more restful since I already had /thoughts as my base endpoint.

I also chose to do a post request to update the likes since I am not sending any information to the server, if I were to update my previous thought I would use put to send in information to the server and update the previously posted information.

### CODE REVIEW

**Comment by malin798** [comment]
forgot the link:
https://malins-happy-thoughts-my-api.netlify.app/

File: Models/Models.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+export const Thought = mongoose.model('Thought', {
+  message: {
+    type: String,
+    minlength: 5,
+    maxlength: 140
+  },
+  name: {
+    type: String,
+    default: "Anonymous"
+  },
```
Comment (KarolCislowski) [inline Models/Models.js:12]: Nice! new field :)

File: server.js [hunk 2]
```diff
@@ -2,28 +2,76 @@ import express from 'express'
...
+    }
+  }
+    
+  try {
+    const thoughts = await Thought.find().limit(20).sort( myFilter ) 
+    res.status(200).json(thoughts)
+  } catch(err) {
+    res.status(404).send(err)
+  }
 })
```
Comment (KarolCislowski) [inline server.js:38]: according to documentation `get "/"` need to return 20 thoughts,

File: server.js [hunk 3]
```diff
@@ -2,28 +2,76 @@ import express from 'express'
...
+app.post('/thoughts', async (req, res) => {
+  try {
+    await new Thought({ message: req.body.message, name: req.body.name, createdAt: new Date().toISOString()}).save()
+    return res.status(200).json(req.body)
+  } catch(err) {
+    res.status(404).send(err)
+  }
+  
+})
+
```
Comment (KarolCislowski) [inline server.js:49]: according to documentation `post "/"` need to create a new one.

File: server.js [hunk 4]
```diff
@@ -2,28 +2,76 @@ import express from 'express'
...
+app.post('/thoughts/:thoughtId/like', async (req, res) => {
+  try {
+    const { thoughtId } = req.params
+    await Thought.updateOne({ _id: thoughtId }, { $inc: { hearts: 1 }})
+    res.status(200).json("update successful")
+  } catch(err) {
+    res.status(404).send(err)
+  }
+})
```
Comment (KarolCislowski) [inline server.js:58]: This one should be `/:thoughtId/like`

**Comment by malin798** [comment]
Ok, I changed it!

**Pull request closed without merging**